pipenv.txt

```diff
@@ -1 +1 @@
-pipenv==11.1.3
+pipenv==10.1.2
```

Comment:
Why are we rolling this back?

Reply:
With 11.1.3, I was getting errors as I removed --skip-lock and --deploy.  I didn't want to argue with Kenneth in the issue I filed, but he attested it should work...

Reply:
@davehunt see e.g. https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/go-bouncer.adhoc/152/console